server/app/schema.py

```diff
@@ -1336,6 +1344,24 @@ def mutate(self, info, **kwargs):
         return EditBuildSettings(user_setting=user_setting)
 
 
+class ChangeProfilePicture(graphene.Mutation):
+    class Arguments:
+        picture = graphene.String(required=True)
+
+    ok = graphene.Boolean(required=True)
+    user = graphene.Field(User)
+
+    def mutate(self, info, **kwargs):
+
+        picture = kwargs.get("picture")
+        with session_scope() as db_session:
+            curr_user = current_user._get_current_object()
+            if not current_user.is_authenticated:
+                raise GraphQLError(_("You are not logged in."))
+            curr_user.profile_picture = picture
+            return ChangeProfilePicture(ok=True, user=curr_user)
```

Comment:
Do we still need ok?

Reply:
I used it in the modal, preferred a nice ok, haha. But I guess I could check for success with `data?.changeProfilePicture?.user?.profilePicture`